iog-ui/app/dashboard/validate-dataset/page.tsx

```diff
@@ -6,8 +6,8 @@ import { useToast } from '@/components/ui/use-toast';
 import { Loader2, UploadCloud } from "lucide-react";
 
 // Define the endpoint URLs
-const SCHEMAS_ENDPOINT = '/api/schemas'; 
-const VALIDATE_ENDPOINT = '/api/validate'; 
+const SCHEMAS_ENDPOINT = '/api/schemas';
+const VALIDATE_ENDPOINT = '/api/validate';
```

Comment:
if these are pointing toward the backend, they don't have `/api` at the start. And the validation endpoint is `/schemas/validate`

Reply:
Thanks for the review. Fixing that